### ISSUE TITLE
GuixSD: Next do not load XDG init.lisp configuration

### ISSUE THREAD

**Issue by alexanderbarbosa** [atlas-engineer/next#220]
Hi, 

Next is loading some other config, default one I presume, instead of the init.lisp under XDG_CONFIG_HOME (~/.config/next/init.lisp)

``` lisp 
<INFO> [09:12:29] next base.lisp (load-lisp-file form-fun-4) -
  Loading configuration from /gnu/store/ygq9p3as7qaxjlil8na66zn4k5yczgzv-sbcl-next-1.2.2-lib/.config/next/init.lisp...
 <WARN> [09:12:29] next base.lisp (start) -
  swank:*communication-style* is set to NIL, recommended value is :fd-handler
```

GuixSD: >1.0 
sblc-next: 1.2.2

**Comment by Ambrevar** [comment]
I can reproduce, this is a mistake of mine!  Will fix as soon as possible!

**Comment by Ambrevar** [comment]
Fixed in 93b26abe0abc3392305c535faf70d0125a2ff786.  Feel free to re-open if the issue persists.
